lib/dfe/reference_data/degrees/subjects.rb

```diff
@@ -1,5480 +1,6976 @@
             dttp_id: '3f8070f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100217' },
           '418670f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Quantum theory and applications',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '418670f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '101300' },
           '498270f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Quarrying',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '498270f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100566' },
           '4d8570f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Quaternary studies',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '4d8570f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '101091' },
           '2d8570f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Radiation physics',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '2d8570f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '101074' },
           '698470f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Radio production',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '698470f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100924' },
           '638470f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Radio studies',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '638470f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100921' },
           'b97f70f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Radiology',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: 'b97f70f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100131' },
           'db8670f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Rail vehicle engineering',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: 'db8670f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '101398' },
           'e57f70f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Railway engineering',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: 'e57f70f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100157' },
           '418070f0-5dce-e911-a985-000d3ab79618' =>
           { name: 'Real estate',
-            synonyms: [],
+            suggestion_synonyms: [],
+            match_synonyms: [],
             dttp_id: '418070f0-5dce-e911-a985-000d3ab79618',
         
```

Comment:
Needs updating with new attribute name to match docs (same for the others below).

```suggestion
            subject_ids: ['937f70f0-5dce-e911-a985-000d3ab79618', # Accounting
```